lib/game.rb

```diff
@@ -0,0 +1,172 @@
+class Game
+  attr_accessor :board,:player_1,:player_2
+
+WIN_COMBINATIONS_GAME = [
+  [0,1,2],
+  [3,4,5],
+  [6,7,8],
+  [0,3,6],
+  [1,4,7],
+  [2,5,8],
+  [0,4,8],
+  [6,4,2]
+  ]
+
+  def initialize(player_1 = Human.new("X"),player_2 = Human.new("O"),board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+  
+  def player_1
+    @player_1
+  end
+
+  def player_2
+    @player_2
+  end
+
+  def current_player
+    @board.turn_count % 2 == 0 ? @player_1 : @player_2
```

Comment:
Thanks Randall. The` % 2 == 0 ` got stuck with me from my beginner labs and exercises. Lol.